### PR TITLE
HA HDFS config

### PR DESCRIPTION
**Pull request by rnpridgeon**
added notes for properly configuring hdfs-connect for HA hdfs.

### CODE REVIEW

File: docs/hdfs_connector.rst [hunk 1]
```diff
@@ -152,6 +152,9 @@ specifies the HDFS we are writing data to and you should set this according to y
 The ``flush.size`` specifies the number of records the connector need to write before invoking file
 commits.
 
+.. note:: For HA HDFS deployments you will need to include ``hadoop.conf.dir``, setting it to a directory which includes both hdfs-site.xml. Once hdfs-site.xml is in place and ``hadoop.conf.dir`` has been set ``hdfs.url`` may be set to the namenodes nameservice id. i.e. 'nameservice1' .
```
Comment (Ishiihara) [inline docs/hdfs_connector.rst]: both is not needed.

**Comment by Ishiihara** [comment]
@rnpridgeon There were several threads in CP mailing list discussed this issue before. e.g. https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/confluent-platform/wKa6gC32Gwo/lBdtn02CKAAJ. Maybe worthwhile adding this to the KB article as well.

File: docs/hdfs_connector.rst (hunk 1 above)
Comment (rnpridgeon) [inline docs/hdfs_connector.rst]: Yep sorry, the 'both' was still there from when I thought I needed core-site.xml as well. Fixing it now

**Comment by rnpridgeon** [comment]
Thanks @Ishiihara, adding a KB around this is definitely on the todo list as well.

**Comment by Ishiihara** [comment]
LGTM. Can you squash before merge? Thanks1

**Comment by rnpridgeon** [comment]
Hi robot, 
[clabot:check]

**Pull request merged**